cmd/lingo/main.go

```diff
@@ -85,18 +61,77 @@ func run() error {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	var cfg struct {
+		Namespace string `env:"NAMESPACE, default=default"`
+
+		// Concurrency per replica.
+		Concurrency int `env:"CONCURRENCY, default=100"`
+
+		ScaleDownDelay int `env:"SCALE_DOWN_DELAY, default=30"`
+		BackendRetries int `env:"BACKEND_RETRIES, default=1"`
+
+		// MessengerURLs is a list of (comma-separated) URLs to listen for requests and send responses on.
+		//
+		// Format: <request-subscription1>|<response-topic1>,<request-subscription2>|<response-topic2>,...
+		//
+		// URL Examples:
+		//
+		// Google PubSub:
```

Comment:
I would like to see an example in this format
"gcppubsub://projects/my-project/subscriptions/my-subscription|gcppubsub://projects/myproject/topics/mytopic"

Reply:
Agreed

Reply:
Done